### PR TITLE
enable GitHub action

### PR DESCRIPTION
**Pull request by mingmin-ddex**

### CODE REVIEW

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Call Kalm Deploy Webhook
+
+on: [push]
```
Comment (davidqhr) [inline .github/workflows/deploy.yaml:3]: Master only?

File: .github/actions/deploy-action.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: 'Deploy to Kalm'
```
Comment (davidqhr) [inline .github/actions/deploy-action.yaml:1]: Should we create a actions repo like https://github.com/actions?

File: .github/workflows/deploy.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    name: A job to deploy new Kalm Component
+    steps:
+      # To use this repository's private action,
+      # you must check out the repository
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Use Deploy Action
+        uses: ./.github/actions # Uses an action in the directory
+        with:
+          KALM_API_ADDRESS: https://dashboard.kapp.live
```
Comment (davidqhr) [inline .github/workflows/deploy.yaml:21]: This value should also put into secret

File: .github/actions/deploy-action.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  KALM_APP:
+    description: ''
+    required: true
+  KALM_COMPONENT:
+    description: ''
+    required: true
+  KALM_COMPONENT_IMG_TAG:
+    description: ''
+    required: true
+# outputs:
```
Comment (davidqhr) [inline .github/actions/deploy-action.yaml:19]: Should we display some useful result that is returned from webhook api?

File: .github/actions/deploy-action.yaml (hunk 2 above)
Comment (mingmin-ddex) [reply .github/actions/deploy-action.yaml]: add a repo for action: https://github.com/kalmhq/kalm-deploy-action

https://github.com/marketplace/actions/kalm-deploy-action

File: .github/workflows/deploy.yaml (hunk 1 above)
Comment (mingmin-ddex) [reply .github/workflows/deploy.yaml]: updated

File: .github/actions/deploy-action.yaml (hunk 4 above)
Comment (mingmin-ddex) [reply .github/actions/deploy-action.yaml]: add response as outputs

File: .github/workflows/deploy.yaml (hunk 3 above)
Comment (mingmin-ddex) [reply .github/workflows/deploy.yaml]: updated

**Comment by mingmin-ddex** [comment]
@davidqhr  updated, plz check again

**Pull request merged**